gobblin-data-management/src/main/java/gobblin/data/management/conversion/hive/extractor/HiveConvertExtractor.java

```diff
@@ -51,12 +57,18 @@
  * to get the corresponding hive {@link org.apache.hadoop.hive.ql.metadata.Table} and hive {@link org.apache.hadoop.hive.ql.metadata.Partition}
  * </p>
  */
+@Slf4j
 public class HiveConvertExtractor implements Extractor<Schema, QueryBasedHiveConversionEntity> {
 
   private List<QueryBasedHiveConversionEntity> conversionEntities = Lists.newArrayList();
 
   public HiveConvertExtractor(WorkUnitState state, FileSystem fs) throws IOException, TException, HiveException {
 
+    if (Boolean.valueOf(state.getPropAsBoolean(PartitionLevelWatermarker.IS_WATERMARK_WORKUNIT_KEY))) {
+      log.info(String.format("Ignoring Watermark workunit for %s", state.getProp(ConfigurationKeys.DATASET_URN_KEY)));
```

Comment:
nit: log.info("Ignoring Watermark workunit for %s", state.getProp(ConfigurationKeys.DATASET_URN_KEY)) achieves the same.